deps/url.lua

```diff
@@ -25,22 +25,28 @@ function exports.parse(url, parseQueryString)
   local chunk, protocol = url:match("^(([a-z0-9+]+)://)")
   url = url:sub((chunk and #chunk or 0) + 1)
 
-  local chunk, auth = url:match('(([0-9a-zA-Z]+:?[0-9a-zA-Z]+)@)') 
+  chunk, auth = url:match('(([0-9a-zA-Z]+:?[0-9a-zA-Z]+)@)') 
```

Comment:
I believe @rphillips used vim for a while and probably knows how to add luacheck to it.